cmd/pebble/cmd_restart.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2014-2021 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"github.com/canonical/pebble/client"
+	"github.com/jessevdk/go-flags"
```

Comment:
Nit: like Juju, we've been separating imports into three groups: stdlib, 3rd party (jessevdk/go-flags), and Pebble (canonical/pebble/client).